package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ajv-moment",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
can you leave this out? releases are handled outside the PR process

Reply:
ok I will change this

Reply:
hi, it should be ok now :)